src/plugins/intel_cpu/src/nodes/rdft.cpp

```diff
@@ -0,0 +1,1753 @@
+        }
+
+        for (size_t K = 0; K < output_size / simd_size; K++) {
+            for (size_t n = 0; n < input_size; n++) {
+                if (type == real_to_complex) {
+                    for (size_t k = 0; k < simd_size; k++) {
+                        twiddles.push_back(std::cos(2 * PI * (K * simd_size + k) * n / input_size));
+                        twiddles.push_back(-std::sin(2 * PI * (K * simd_size + k) * n / input_size));
+                    }
+                } else if (type == complex_to_real) {
+                    for (size_t k = 0; k < simd_size; k++) {
+                        twiddles.push_back(std::cos(2 * PI * (K * simd_size + k) * n / input_size));
+                    }
+                    for (size_t k = 0; k < simd_size; k++) {
+                        twiddles.push_back(-std::sin(2 * PI * (K * simd_size + k) * n / input_size));
+                    }
+                } else if (type == complex_to_complex) {
+                    for (size_t k = 0; k < simd_size; k++) {
+                        twiddles.push_back(std::cos(2 * PI * (K * simd_size + k) * n / input_size));
+                        twiddles.push_back(std::cos(2 * PI * (K * simd_size + k) * n / input_size));
+                    }
+                    for (size_t k = 0; k < simd_size; k++) {
+                        twiddles.push_back(-std::sin(2 * PI * (K * simd_size + k) * n / input_size));
+                        twiddles.push_back(-std::sin(2 * PI * (K * simd_size + k) * n / input_size));
+                    }
+                }
+            }
+        }
+        if ((output_size % simd_size) != 0) {
+            size_t start = (output_size / simd_size) * simd_size;
+            for (size_t k = start; k < output_size; k++) {
+                for (size_t n = 0; n < input_size; n++) {
+                    twiddles.push_back(std::cos(2 * PI * k * n / input_size));
+                    twiddles.push_back(-std::sin(2 * PI * k * n / input_size));
+                }
+            }
+        }
+        return twiddles;
+    }
+
+    void dft(float* input_ptr, float* twiddles_ptr, float* output_ptr,
+             size_t input_size, size_t signal_size, size_t output_size,
+             enum dft_type type, bool parallelize) override {
+        jit_dft_kernel* kernel = type == complex_to_complex ? dft_kernel.get() : rdft_kernel.get();
+        if (parallelize) {
+            const int cacheline_size = 64;
+            size_t block_size = 4 * cacheline_size / sizeof(float);
+            size_t num_blocks = (output_size + block_size - 1) / block_size;
+            parallel_nt(num_blocks, [&] (size_t i, size_t nthr) {
+                if (num_blocks > nthr) {
+                    auto new_block_size = (((output_size / nthr) + block_size - 1) / block_size) * block_size;
+                    block_size = new_block_size;
+                    num_blocks = nthr;
+                }
+                jit_dft_args args {
+                    .input = input_ptr,
```

Comment:
We support just C++11 compiler version now. Looks like such syntax is not supported by this compiler.